### PR TITLE
AELO year 3 updates to run all IMTs

### PR DESCRIPTION
**Pull request by AnnaRood**
Updated AELO Year 1 IMTs [PGA, SA(0.2), and SA(1.0)] to run Year 3 [PGA, SA(0.02) -> SA(10)].
[Note: no SA(0) yet]

Updated to ASCE7-22 parameters.

Plots updated to show:
- One plot for each IMT for the disagg by src (for us) 
- 3-piece disagg plot kept as PGA, 0.2 s , 1.0 s
- Mean hazard curves for a subset of IMTs, and moved from maximum component to geometric mean hazard curves.
- MCE results for all periods, and two subplots for a log and linear scale

### CODE REVIEW

File: openquake/calculators/postproc/compute_rtgm.py [hunk 1]
```diff
@@ -85,12 +85,17 @@
 # NOTE: for meanHCs_afe_RTGM and disaggr_by_src we want to display these
 # three imts, that are mandatory in this context. For the plot of governing
 # MCE we read imts from the imtls
-IMTS = ['PGA', 'SA(0.2)', 'SA(1.0)']
+IMTS = ['PGA', 'SA(0.02)', 'SA(0.03)', 'SA(0.05)', 'SA(0.075)', 'SA(0.1)',
+        'SA(0.15)', 'SA(0.2)', 'SA(0.25)', 'SA(0.3)', 'SA(0.4)', 'SA(0.5)',
+        'SA(0.75)', 'SA(1.0)', 'SA(1.5)', 'SA(2.0)', 'SA(3.0)', 'SA(4.0)',
+        'SA(5.0)', 'SA(7.5)', 'SA(10.0)']
```
Comment (micheles) [inline openquake/calculators/postproc/compute_rtgm.py:91]: I thought most GMPEs could not be extended to SA(10.0) and we decided to discard it (?)

Comment (AnnaRood) [reply openquake/calculators/postproc/compute_rtgm.py]: We have now updated so that the IMTS are defined in the job file, instead of using defaults

**Review by kejohnso — APPROVED** [review]
LGTM!

File: openquake/calculators/postproc/compute_rtgm.py [hunk 2]
```diff
@@ -115,7 +110,9 @@ def calc_rtgm_df(hcurves, site, site_idx, oq):
     :returns: pandas dataframe with RTGM and related parameters
     :returns: numpy array of conversion factors to max component
     """
+    IMTS = list(oq.imtls)
```
Comment (micheles) [inline openquake/calculators/postproc/compute_rtgm.py:113]: The conventions is to use all caps for constants, so IMTS is a bad name. You can use `imts` or `job_imts` or `orig_imts` or whatever you like more.

**Comment by micheles** [comment]
Please update the changelog. We have released engine 3.20 so now this branch can be merged.

**Review by micheles — APPROVED** [review]

**Comment by micheles** [comment]
Thanks to Matteo, now we cannot forget to add people to CONTRIBUTORS.txt! (see the error in the documentation tests ;-))

**Pull request merged**